contracts/src/tokens/lock/LockBase.sol

```diff
@@ -24,31 +25,25 @@ abstract contract LockBase is ILockBase {
   function _enableLock(address caller) internal {
     LockStorage.Layout storage ds = LockStorage.layout();
 
-    if (ds.enabledByAddress[caller]) {
-      revert LockAlreadyEnabled();
-    }
+    if (ds.enabledByAddress[caller])
+      CustomRevert.revertWith(LockAlreadyEnabled.selector);
 
     ds.enabledByAddress[caller] = true;
 
-    emit LockUpdated(caller, true, 0, block.timestamp);
+    emit LockUpdated(caller, true, 0);
```

Comment:
Why remove block.timestamp ? Also, are we sure nothing is depending on this current event signature that will break after this pr ?

Reply:
Events can only be accessed offchain. When queried offchain, event logs come with meta data such as block number and contract address already.
https://github.com/wevm/viem/blob/afd1f787e2f69f599cc5cd943e28b0c39fe016c9/src/types/log.ts#L15

Nothing is using the event in the codebase.